sdk/postgresql/Azure.ResourceManager.PostgreSql/api/Azure.ResourceManager.PostgreSql.netstandard2.0.cs

```diff
@@ -793,19 +1218,30 @@ public partial class PostgreSqlFlexibleServerBackupProperties
         public System.DateTimeOffset? EarliestRestoreOn { get { throw null; } }
         public Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexibleServerGeoRedundantBackupEnum? GeoRedundantBackup { get { throw null; } set { } }
     }
-    public partial class PostgreSqlFlexibleServerCapabilityProperties
+    public partial class PostgreSqlFlexibleServerBackupSettings
+    {
+        public PostgreSqlFlexibleServerBackupSettings(string backupName) { }
+        public string BackupName { get { throw null; } }
+    }
+    public partial class PostgreSqlFlexibleServerBackupStoreDetails
     {
-        internal PostgreSqlFlexibleServerCapabilityProperties() { }
-        public bool? FastProvisioningSupported { get { throw null; } }
-        public bool? IsGeoBackupSupported { get { throw null; } }
-        public bool? IsZoneRedundantHAAndGeoBackupSupported { get { throw null; } }
-        public bool? IsZoneRedundantHASupported { get { throw null; } }
-        public string Status { get { throw null; } }
+        public PostgreSqlFlexibleServerBackupStoreDetails(System.Collections.Generic.IEnumerable<string> sasUriList) { }
+        public System.Collections.Generic.IList<string> SasUriList { get { throw null; } }
+    }
+    public partial class PostgreSqlFlexibleServerCapability : Azure.ResourceManager.PostgreSql.FlexibleServers.Models.CapabilityBase
```

Comment:
There are operations that previously return `PostgreSqlFlexibleServerCapabilityProperties` and now return this type.
This is breaking change, we need to revert it back.
We could do this simply by rename this type back to `PostgreSqlFlexibleServerCapabilityProperties`.
I also saw some properties in this type changed as well which might also be breaking changes but this is not reported until we could change the type name back.

Reply:
Renamed the class, not to sure if we need to fix for properties too